<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLabControlPanelClass</class>
 <widget class="QMainWindow" name="GLabControlPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabExe</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Workbenches</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QListWidget" name="pListWorkBenches"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pButtonCreateNewWorkBench">
           <property name="text">
            <string>New workbench</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pWorkBenchFromFile">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>From file</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Devices</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pButtonDeviceShelf">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Device shelf</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pButtonProgDeviceShelf">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>e-Devices</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveAllWorkbenches"/>
    <addaction name="actionSave_all_as"/>
    <addaction name="actionExport_wb_Screenshots"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionCheck_for_update"/>
   </widget>
   <widget class="QMenu" name="pMenuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="pMenuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="pMainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="pStatusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="pDebugDock">
   <property name="windowTitle">
    <string>Debuging informations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextEdit" name="pDebugTextEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="pDevicesComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveAllWorkbenches">
   <property name="text">
    <string>Save all workbenches</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This saves all the workbenches in the list of workbenches.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionFind_update">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionSave_all_as">
   <property name="text">
    <string>Save all as ...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save all the workbenches and the main configuration file in the folder of your choice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionExport_wb_Screenshots">
   <property name="text">
    <string>Export wb Screenshots</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exports all the active workbenches as png screenshots in a tree structure of sub-folders created in the main configuration directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
